modules/nw-rfhe-creating_bmc_event_sub.adoc

```diff
@@ -0,0 +1,105 @@
+// Module included in the following assemblies:
+//
+// * networking/using-rfhe.adoc
+
+:_content-type: PROCEDURE
+[id="nw-rfhe-creating_bmc_event_sub_{context}"]
+= Subscribing to Redfish hardware events
+
+You can configure applications that are running on bare-metal cluster nodes to subscribe to Redfish hardware events that the baseboard management controller (BMC) generates. Example Redfish events include increase in device temperature, or removal of a device. Applications subscribe to Redfish hardware events using a REST API.
+
+Before you can subscribe to Redfish hardware events for the node, you must create a `BMCEventSubscription` custom resource (CR) for the node.
+
+[IMPORTANT]
+====
+You can only create a `BMCEventSubscription` CR for physical hardware that supports Redfish and has a vendor interface set to `redfish` or `idrac-redfish`.
+====
+
+[NOTE]
+====
+Use the `BMCEventSubscription` CR to subscribe to pre-defined Redfish events. The Redfish standard does not provide an option to create specific alerts and thresholds. For example, to receive an alert event when an enclosure's temperature exceeds 40° Celsius, you need to manually configure the event according to the vendor's recommendations.
+====
+
+.Prerequisites
+
+* Install the OpenShift CLI (`oc`).
+* Log in as a user with `cluster-admin` privileges.
+* Deploy a bare-metal node with Redfish hardware in your cluster.
+* Get the user name and password for the bare-metal node BMC.
+
+.Procedure
+
+. Confirm the hardware has the Redfish `EventService` enabled using the following `curl` command:
++
+[source,terminal]
+----
+curl https://<bmc_ip_address>/redfish/v1/EventService --insecure -H 'Content-Type: application/json' -u "<user_name>:<password>"
+----
++
+where:
++
+--
+bmc_ip_address:: is the IP address of the BMC controller
+--
++
+.Example output
+[source,terminal]
+----
+{
+   "@odata.context":"/redfish/v1/$metadata#EventService.EventService",
+   "@odata.id":"/redfish/v1/EventService",
+   "@odata.type":"#EventService.v1_0_2.EventService",
+   "Actions":{
+      "#EventService.SubmitTestEvent":{
+         "EventType@Redfish.AllowableValues":[
+            "StatusChange",
+            "ResourceUpdated",
+            "ResourceAdded",
+            "ResourceRemoved",
+            "Alert"
+         ],
+         "target":"/redfish/v1/EventService/Actions/EventService.SubmitTestEvent"
+      }
+   }
+...
+}
```

Comment:
Maybe I'm a bit lost, but the output above should have the `"ServiceEnabled": true,` this would be the information the person should look at to understand if the service is enabled.

Reply:
Added this.